docs/csharp/language-reference/operators/lambda-expressions.md

```diff
@@ -44,44 +44,57 @@ When you use method-based syntax to call the <xref:System.Linq.Enumerable.Select
   
 ## Expression lambdas
 
-A lambda expression with an expression on the right side of the `=>` operator is called an *expression lambda*. Expression lambdas are used extensively in the construction of [expression trees](../../programming-guide/concepts/expression-trees/index.md). An expression lambda returns the result of the expression and takes the following basic form:
+A lambda expression with an expression on the right side of the `=>` operator is called an *expression lambda*. An expression lambda returns the result of the expression and takes the following basic form:
 
 ```csharp
 (input-parameters) => expression
 ```
 
-The parentheses are optional only if the lambda has one input parameter; otherwise they are required.
+The body of an expression lambda can consist of a method call. However, if you are creating [expression trees](../../programming-guide/concepts/expression-trees/index.md) that are evaluated outside the context of the .NET common language runtime, such as in SQL Server, you should not use method calls in lambda expressions. The methods will have no meaning outside the context of the .NET common language runtime.
 
-Specify zero input parameters with empty parentheses:  
+## Statement lambdas
+
+A statement lambda resembles an expression lambda except that the statement(s) is enclosed in braces:
+
+```csharp  
+(input-parameters) => { <sequence-of-statements> }
+```
+
+The body of a statement lambda can consist of any number of statements; however, in practice there are typically no more than two or three.
+
+[!code-csharp-interactive[statement lambda](snippets/lambda-expressions/ExpressionAndStatementLambdas.cs#StatementLambda)]
+
+You cannot use statement lambdas to create expression trees.
+
+## Input parameters of a lambda expression
```

Comment:
That information was inside the section about expression lambdas, while it's true for all lambdas. So, I've extracted the section.